tests/fixtures/locator.spdx.json

```diff
@@ -0,0 +1,162 @@
+            "originator": "Person: GitHub Inc.",
+            "externalRefs": [
+                {
+                    "referenceCategory": "PACKAGE-MANAGER",
+                    "referenceType": "npm",
+                    "referenceLocator": "%40npmcli/fs@2.1.2"
+                }
+            ]
+        },
+        {
+            "SPDXID": "SPDXRef-Package-go-module-gopkg.in-yaml.v2-538547a367f862a3",
+            "name": "gopkg.in/yaml.v2",
+            "downloadLocation": "NOASSERTION",
+            "licenseConcluded": "NONE",
+            "licenseDeclared": "NONE",
+            "copyrightText": "NOASSERTION",
+            "versionInfo": "v2.3.0",
+            "sourceInfo": "acquired package info from go module information: /root/go/pkg/mod/github.com/imdario/mergo@v0.3.12/go.mod",
+            "externalRefs": [
+                {
+                    "referenceCategory": "PACKAGE-MANAGER",
+                    "referenceType": "golang",
+                    "referenceLocator": "gopkg.in/yaml.v2@v2.3.0"
+                }
+            ]
+        },
+        {
+            "SPDXID": "SPDXRef-Package-python-async-timeout-62f2f5d69566701a",
+            "name": "async-timeout",
+            "downloadLocation": "NOASSERTION",
+            "licenseConcluded": "LicenseRef-Apache-2",
+            "licenseDeclared": "LicenseRef-Apache-2",
+            "copyrightText": "NOASSERTION",
+            "versionInfo": "4.0.2",
+            "sourceInfo": "acquired package info from installed python package manifest file: /usr/local/lib/python3.9/dist-packages/async_timeout-4.0.2.dist-info/METADATA, /usr/local/lib/python3.9/dist-packages/async_timeout-4.0.2.dist-info/RECORD, /usr/local/lib/python3.9/dist-packages/async_timeout-4.0.2.dist-info/top_level.txt",
+            "originator": "Person: Andrew Svetlov <andrew.svetlov@gmail.com> (andrew.svetlov@gmail.com)",
+            "externalRefs": [
+                {
+                    "referenceCategory": "PACKAGE-MANAGER",
+                    "referenceType": "pypi",
+                    "referenceLocator": "async-timeout@4.0.2"
+                }
+            ]
+        },
+        {
+            "SPDXID": "SPDXRef-Package-dotnet-Newtonsoft.Json-4655a9695632e63c",
+            "name": "Newtonsoft.Json",
+            "downloadLocation": "NOASSERTION",
+            "licenseConcluded": "NONE",
+            "licenseDeclared": "NONE",
+            "copyrightText": "NOASSERTION",
+            "versionInfo": "13.0.1",
+            "sourceInfo": "acquired package info from dotnet project assets file: /root/.dotnet/sdk/7.0.202/DotnetTools/dotnet-user-secrets/7.0.4-servicing.23119.3/tools/net7.0/any/dotnet-user-secrets.deps.json",
+            "externalRefs": [
+                {
+                    "referenceCategory": "PACKAGE-MANAGER",
+                    "referenceType": "nuget",
+                    "referenceLocator": "Newtonsoft.Json@13.0.1"
```

Comment:
This doesn't match the [spec][1]... Shouldn't this be `"Newtonsoft.Json/13.0.1"`?

[1]: https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f33-nuget